docker-compose.yaml

```
@@ -0,0 +1,26 @@
+version: '3.8'
+
+services:
+  service:
+    build: .
+    command: poetry run gunicorn src.main:app --workers 4 --worker-class uvicorn.workers.UvicornWorker --bind 0.0.0.0:8000
+    ports:
+      - 8000:8000
+    environment:
+      - DATABASE_URI=postgresql://${DB_USER}:${DB_PASS}@${DB_HOST}:${DB_PORT}/${DB_NAME}
+    depends_on:
+      - db
+
```

```suggestion
    env_file:
      - .env
```
В дальнейшем мы будем использовать environment variables, типа DEBUG, LOG_LEVE и далее, чтобы каждый раз compose не менять, лучше указать ссылку на файл со всеми переменными
https://docs.docker.com/compose/environment-variables/set-environment-variables/#use-the-environment-attribute